warehouse/integrations/snowflake/snowflake_test.go

```diff
@@ -1258,6 +1258,99 @@ func TestIntegration(t *testing.T) {
 		require.NoError(t, rows.Err())
 		require.Empty(t, ids2, "no more rows left")
 	})
+
+	t.Run("Add Columns", func(t *testing.T) {
```

Comment:
Please add a test scenario where some columns are already present and some are new columns.